www/css/mod.css

```diff
@@ -26,3 +26,8 @@
 .mdl-js-data-table {
   width: 100%;
 }
+
+/* This makes the login and register boxes look right. We'll fix this later */
```

Comment:
if this is for the login and register boxes, then let's not apply it to all `p` tags, can we use a classname?